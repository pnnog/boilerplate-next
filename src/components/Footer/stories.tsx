import { Meta, Story } from '@storybook/react';

import Footer from '.';

export default {
  title: 'Footer',
  component: Footer,
  args: {
    title: 'Título padrão para todos os stories caso nada seja passado',
    description:
      'Description padrão para todos os stories caso nada seja passado'
  }
} as Meta;

export const Basic: Story = (args) => <Footer {...args} />;
