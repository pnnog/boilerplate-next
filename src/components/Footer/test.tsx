import { screen } from '@testing-library/react';
import { renderWithTheme } from 'utils/tests/helper';

import Footer from '.';

describe('<Footer />', () => {
  it('should render heading', () => {
    const { container } = renderWithTheme(<Footer />);

    expect(
      screen.getByRole('heading', { name: /Footer/i }).parentNode
    ).toHaveStyle({ background: '#FFFFFF' });

    expect(container.firstChild).toMatchSnapshot();
  });
});
