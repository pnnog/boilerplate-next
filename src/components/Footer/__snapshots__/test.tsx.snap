// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render heading 1`] = `
.c0 {
  background: #FFFFFF;
}

<div
  class="c0"
>
  <h1>
     Footer 
  </h1>
</div>
`;
